Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM       node:16 as builder
-RUN        npm i -g --unsafe-perm @subql/node@0.23.1
+USER       node
+RUN        npm cache clean \ 
```

Comment:
Check the size of image in 3 cases:
1. before your changes
2. after your changes
3. change order (put cache pruning after installation `npm cache clean --force `)
let me know

Reply:
1. after my changes: 133MB
2. before (dev image):  290MB
3. npm cache clean must be before installation because otherwise the node user doesn't have enough permissions
https://docker.soramitsu.co.jp/harbor/projects/16/repositories/sora2%2Fsubquery